MetaMorpheus/EngineLayer/PsmTsv/PsmFromTsv.cs

```diff
@@ -126,6 +128,7 @@ public PsmFromTsv(string line, char[] split, Dictionary<string, int> parsedHeade
             }
 
             PrecursorCharge = (int)double.Parse(spl[parsedHeader[PsmTsvHeader.PrecursorCharge]].Trim(), CultureInfo.InvariantCulture);
+            PrecursorIntensity = (parsedHeader[PsmTsvHeader.PrecursorIntensity] < 0) ? null : (double?)double.Parse(spl[parsedHeader[PsmTsvHeader.PrecursorIntensity]].Trim(), CultureInfo.InvariantCulture);
```

Comment:
Would Int.TryParse() work better for this than the current implementation? That way if there is invalid input, the method would return false, and you can handle the invalid input from there.